sentry_sdk/integrations/django/caching.py

```diff
@@ -0,0 +1,101 @@
+import functools
+from typing import TYPE_CHECKING
+
+from django import VERSION as DJANGO_VERSION
+from django.core.cache import CacheHandler
+
+from sentry_sdk import Hub
+from sentry_sdk.consts import OP, SPANDATA
+from sentry_sdk._compat import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Callable
+
+
+METHODS_TO_INSTRUMENT = [
+    "get",
+    "get_many",
+]
+
+
+def _instrument_call(cache, method_name, original_method, args, kwargs):
+    # type: (CacheHandler, str, Callable[..., Any], Any, Any) -> Any
+    description = "{} {}".format(method_name, " ".join(args))
```

Comment:
what does `args` look like here? Can you post a sample screenshot maybe?

Reply:
It is (mostly, I guess) just the cache key: https://sentry-sdks.sentry.io/performance/django-otel:14130998f01b457799bf724fbbcc9a74/?project=6539771&query=http.method%3AGET&referrer=performance-transaction-summary&showTransactions=recent&statsPeriod=14d&transaction=%2F&unselectedSeries=p100%28%29

Reply:
if we later instrument more than just the cache getter it could also be the value that needs to be cached, then we need to check for PII stuff.